pkg/blockstorage/azure/client_test.go

```diff
@@ -87,21 +86,23 @@ func (s ClientSuite) TestGetRegions(c *C) {
 
 func (s *ClientSuite) TestGetCredConfig(c *C) {
 	for _, tc := range []struct {
-		env        azure.Environment
+		name       string
+		env        Environment
 		config     map[string]string
 		errChecker Checker
-		expCCC     auth.ClientCredentialsConfig
+		expCCC     ClientCredentialsConfig
 	}{
 		{
-			env: azure.PublicCloud,
+			name: "TC1",
```

Comment:
[nit] shoudl we have a more descriptive test case name? This would help w triaging test failures through `go test` output . Ditto for the rest of test case names